controllers/grid/settings/genre/form/GenreForm.inc.php

```diff
@@ -61,9 +61,9 @@ function __construct($genreId = null) {
 	/**
 	 * Initialize form data from current settings.
 	 * @param $args array
-	 * @param $request PKPRequest
 	 */
-	function initData($args, $request) {
+	function initData($args) {
```

Comment:
Was `$args` intentionally left here?

Reply:
Yes; it’s referenced at the end of the body of that method. There may be a better way to pass it in, but this satisfies PHP’s warnings.